### ISSUE TITLE
Typo in CGAL-4.6 installer files

### ISSUE THREAD

**Issue by DragonJujo** [CGAL/cgal#146]
In CGAL-4.6/src/CMakelist.txt - This line
```
    string(REPLACE "//CMakeLists.txt" "" CGAL_CONFIGURED_LIBRARY_NAME ${libconfigfile})
```
should be.
```
    string(REPLACE "/CMakeLists.txt" "" CGAL_CONFIGURED_LIBRARY_NAME ${libconfigfile})
```

**Comment by bo0ts** [comment]
This has been addressed by #131

**Comment by lrineau** [comment]
By #132, actually.

The bug is already fixed in CGAL-4.6.1, that was released on Sunday but not yet announce (because of an outage of our hosting service InriaForge).

Fixed by #132.
